src/rust/graph-mutation/src/reverse_edge_resolver.rs

```diff
@@ -36,33 +37,50 @@ impl ReverseEdgeResolver {
         }
     }
 
+    #[tracing::instrument(skip(self), err)]
     pub async fn resolve_reverse_edge(
         &self,
         tenant_id: uuid::Uuid,
         node_type: NodeType,
         edge_name: EdgeName,
     ) -> Result<EdgeName, ReverseEdgeResolverError> {
-        match self
-            .r_edge_cache
-            .entry((tenant_id, edge_name.clone(), node_type.clone()))
-        {
-            dashmap::mapref::entry::Entry::Occupied(entry) => {
-                Ok(entry.get().reverse_edge_name.clone())
-            }
-            dashmap::mapref::entry::Entry::Vacant(entry) => {
-                let mut schema_client = self.schema_client.clone();
-                let response = schema_client
-                    .get_edge_schema(GetEdgeSchemaRequest {
-                        tenant_id,
-                        node_type: node_type.clone(),
-                        edge_name: edge_name.clone(),
-                    })
-                    .await?;
+        let mut schema_client = self.schema_client.clone();
+        let response = schema_client
+            .get_edge_schema(GetEdgeSchemaRequest {
+                tenant_id,
+                node_type: node_type.clone(),
+                edge_name: edge_name.clone(),
+            })
+            .await?;
 
-                let reverse_name = response.reverse_edge_name.clone();
-                entry.insert(response);
-                Ok(reverse_name)
-            }
-        }
+        let reverse_name = response.reverse_edge_name;
+        Ok(reverse_name)
+
+        // TODO: Resurrect the below once we figure out caching for Graph Mutation
```

Comment:
commented code can be removed

Reply:
Fixing these is high on our list of things to do (like, this week), and the uncommented code will likely be exactly the same as what's already here [barring some type changes], so I'm intentionally keeping it as-is. However, I can guarantee I'll resolve these comments by Friday.